lm_eval/models/gpt2.py

```diff
@@ -61,7 +105,7 @@ def __init__(
 
         if isinstance(
```

Comment:
I think this has nothing to do in this class .... this should be in test somewhere. But you should probably check that the tokenizer name is `gpt2` or something so that it passes without having to pass `skip_tokenizer`

Reply:
You can have some niche cases where the tokenizer is also a GPT2 tokenizer but trained differently (eg OPT) that is why I have put this flag